.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -105,10 +105,10 @@ fi
 echo "=== END OF ENVIRONMENT INFORMATION ==="
 ```
 
-# 📝 Additional Context
+## 📝 Additional Context
 
 Include any other information that may help us understand the issue, such as:
 
-- Recent changes to the configuration or code.
-- Whether the issue occurs consistently or intermittently.
-- Any troubleshooting steps you have already tried.
+-                     Recent changes to the configuration or code.
```

Comment:
I don't understand why pre-commit did that...

Reply:
I believe the linter shouldn't even run here. Let's just apply it to `docs/**/*.md` in this iteration, please.